exampleCourse/externalGradingFilesCourse/python_autograder/run.sh

```diff
@@ -19,16 +23,31 @@ mkdir $BIN_DIR
 mkdir $OUT_DIR
 
 cp $STUDENT_DIR* $BIN_DIR
-cp $SHARED_DIR* $MERGE_DIR
+cp $AG_DIR* $MERGE_DIR
 cp $TEST_DIR* $MERGE_DIR
 
-# move our run script into its proper location
-cp $SHARED_DIR'run.sh' $JOB_DIR'run.sh'
-
 # we need this to include code as python modules
 # There is already one in the /run directory, but we need one in the /run/bin directory as well
 echo "" > $BIN_DIR/__init__.py
```

Comment:
After too much time reading the python import docs, it turns out that we do need this line above because we will end up doing an import from the `bin` subdirectory. However, we don't need the file `exampleCourse/questions/fibonacciExternalUpload/tests/__init__.py` because we are importing `autograder.py` from `autograder_wrapper.py` in the same directory.